game/environment.go

```diff
@@ -88,22 +102,68 @@ func NewEnvironment(conf *GameConfig, vlib vbox.Library) (*GameConfig, error) {
 		dockerHost: dockerHost,
 		vlib:       vlib,
 	}
-
-	conf.env = env
-
+	g := &Game{
+		env:    env,
+		config: conf,
+	}
 	log.Info().Msgf("New environment initialized ")
-	return conf, nil
+	return g, nil
+}
+
+func (g *Game) Config() *GConfig {
+	return g.config
 }
 
-func (env *environment) Close() error {
-	//var wg sync.WaitGroup
-	// var closers []io.Closer
+func (g *Game) Close() error {
+	var wg sync.WaitGroup
+	var closeError error
```

Comment:
When doing like this the error is overwritten if multiple errors occur, is there a nice fix for this?